client/src/api/routes/users.ts

```diff
@@ -1,7 +1,22 @@
 import { useQuery, UseQueryResult } from "react-query";
 import { fetcher } from "api/fetcher";
 import { doEmailSignIn, doEmailSignUp, doGoogleSignIn, doSignOut } from "api/firebase/functions";
-import { Failure } from "typings";
+import { Failure, UserObject } from "typings";
+import { AxiosResponse } from "axios";
+
+/**
+ * @description GET /setting
+ * @param {string} userId 
+ * @returns {UseQueryResult<UserObject>} the requested user
+ * @throws if fails
+ */
+export const useGetUser = (userId: string): UseQueryResult<UserObject> => {
```

Comment:
Not sure why this is added here. this would be a dangerous route to expose and it does not exist on the backend anyways

Reply:
I intended to use this for getting a user's info to display on the settings page

Reply:
if anything, make a getUser that does not take in any arguements and then it will simply grab the user who is logged in on the backend part